### ISSUE TITLE
Add support for common non-standard ID3v2 frames

### ISSUE THREAD

**Issue by thebigmunch** [thebigmunch/audio-metadata#27]
Some examples:

* COMM w. desc="iTunNORM": iTunes Normalization settings
* RGAD: Replay Gain Adjustment
* TCMP: iTunes Compilation Flag
* TSO2: iTunes uses this for Album Artist sort order
* TSOC: iTunes uses this for Composer sort order
* XRVA: Experimental RVA2

Related to/dependent on #18.

**Comment by thebigmunch** [comment]
Missed a reference to https://github.com/thebigmunch/audio-metadata/commit/c31bcbd26f8ade325191a1f74c3374eb649f51fa

**Comment by thebigmunch** [comment]
I think I got most of the common ones. If anyone finds more, post a new issue.
